src/event.ts

```diff
@@ -377,6 +380,23 @@ export interface OutputModified extends InfoBase {
   detail: 'output-modified';
 }
 
+/**
+ * The analysis phase for an execution has begun, where we load package.json
+ * files, analyze their wireit configs, and build the dependency graph.
+ */
+export interface AnalysisStarted extends InfoBase {
+  detail: 'analysis-started';
+}
+
+/**
+ * The analysis phase for an execution has begun, where we load package.json
+ * files, analyze their wireit configs, and build the dependency graph.
+ */
```

Comment:
Update to reflect completion of analysis.

Reply:
Good catch! Done